lib/Advisor.js

```diff
@@ -82,9 +81,9 @@ class Advisor {
 
   handleSyncChange() {
     if (Preferences.isSet('services.sync.username')) {
-      this.panel.port.emit('syncEnabled');
-    } else {
-      this.panel.port.emit('syncDisabled');
+      // end study if they sign up for sync
+      simpleStorage.storage.recData = undefined; // clear all recs
```

Comment:
You clear these and uninstall the add-on in more than one place, you should factor this into a function in case the uninstall process changes and you need to update it everywhere.